### ISSUE TITLE
Publish a PS 0.15 compatible release?

### ISSUE THREAD

**Issue by newlandsvalley** [rnons/purescript-html-parser-halogen#20]
Have you any thoughts of creating a PS 0.15-compatible release?  I _think_ I'm right in saying that, now the new registry is operational, it should pick up a new release automatically were you to cut one and publish to pursuit and package-sets.

**Comment by shd101wyy** [comment]
Yeah I got the same problem. So for my project I will have to use purescript 0.13 :joy:

**Comment by newlandsvalley** [comment]
I am currently just using the penultimate commit - 035a51d02ba9f8b70c3ffd9fe31a3f5bed19941c which has moved it to PS 0.15 - it's just that there's been no release.
